src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -18,11 +18,44 @@ import { UrlPathTemplate } from '../../../utils/urlPaths';
 import { authenticatedApiRequest } from '../../../api/apiRequest';
 import { getValueByKey } from '../../../utils/user-helpers';
 import { fetchPerson } from '../../../api/cristinApi';
+import { postOrcidCredentials } from '../../../api/orcidApi';
+import { OrcidCredentials } from '../../../types/orcid.types';
 
 interface UserOrcidProps {
   user: User;
 }
 
+const allPropertiesAreInCredentialsObject = (rawCredentials: { [p: string]: string }) => {
+  return (
+    'expires_in' in rawCredentials &&
+    !isNaN(+rawCredentials['expires_in']) &&
+    'id_token' in rawCredentials &&
+    'persistent' in rawCredentials &&
+    'tokenId' in rawCredentials &&
+    !isNaN(+rawCredentials['tokenId']) &&
+    'tokenVersion' in rawCredentials &&
+    'token_type' in rawCredentials &&
+    'access_token' in rawCredentials
+  );
+};
```

Comment:
Personlig foretrekker jeg inline når man kan, men det blir jo til samme koden uansett så helt greit å la det stå som det er også
```suggestion
const allPropertiesAreInCredentialsObject = (rawCredentials: { [p: string]: string }) => 
  (
    'expires_in' in rawCredentials &&
    !isNaN(+rawCredentials['expires_in']) &&
    'id_token' in rawCredentials &&
    'persistent' in rawCredentials &&
    'tokenId' in rawCredentials &&
    !isNaN(+rawCredentials['tokenId']) &&
    'tokenVersion' in rawCredentials &&
    'token_type' in rawCredentials &&
    'access_token' in rawCredentials
  );
```